src/main/java/co/zecko/retailer/common/pojo/Discount.java

```diff
@@ -1,7 +1,7 @@
 package co.zecko.retailer.common.pojo;
 
 import co.zecko.retailer.common.enums.DiscountType;
-import co.zecko.retailer.common.pojo.base.BaseEntry;
+import co.zecko.common.java.pojo.BaseEntry;
```

Comment:
```suggestion
import co.zecko.common.public.pojo.BaseEntry;
```

Reply:
@akartik80 we can't use public keyword in import